tests/ast/README.md

```diff
@@ -0,0 +1,25 @@
+## AST Tests
+
+This driver enables testing of the AST generation that will be used in the server-side Snowpark implementation, starting with Phase 0.
+
+All generated AST should be tested using this mechanism. To add a test, create a new file under `tests/ast/data`. Files look like the following example. The test driver sets up the session and looks at the accumulated lazy values in the resulting environment.
+
+N.B. No eager evaluation is permitted, as any intermediate batches will not be observed. This can easily be changed if necessary, however.
+
+```python
+## TEST CASE
+
+df = session.table("test_table")
+df = df.filter("STR LIKE '%e%'")
+
+## EXPECTED OUTPUT
```

Comment:
could we make the expected output agnostic to empty lines? Feels like we may have a lot of troubleshooting with empty lines, and I don't think having a mismatch in empty lines warrants a test failure.

Reply:
There's a _super_ easy way to reset these test expectations if the output changes. I don't think doing this is necessary and I would argue that having the exact correct output is cleaner than not.